### PR TITLE
BaSP-MR-17: initial setup complete

### PR DESCRIPTION
**Pull request by IgnacioCanton**
all initial setup finished. please review so I can merge it.

### CODE REVIEW

**Review by LucianoStradiot — APPROVED** [review]

**Review by VictoriaNegrotto — APPROVED** [review]
looks good

**Review by matiasluispeter — APPROVED** [review]
seems good to me!!

**Review by NicolasKler — APPROVED** [review]
Looks good to me

**Review by leo-balonchard — APPROVED** [review]
Looks good

File: src/index.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+  .then(() => console.log('MongoDB connected'))
+  .catch((e) => console.log(e));
 
-app.use(cors());
-app.use(express.json());
-
-const DB_URL = 'mongodb+srv://ferpa-team:sJMD9bZjg8Bq4ntf@megarocket-databases.inpprte.mongodb.net/ferpa-database';
-
-mongoose
-  .connect(DB_URL, { maxPoolSize: process.env.MONGO_POOLSIZE || 1 })
```
Comment (Juanoauil) [inline src/index.js:16]: Please revert this change, you must set with `maxPoolSize`

**Review by Juanoauil — CHANGES_REQUESTED** [review]

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**